rest-api/src/main/java/gov/cms/qpp/conversion/api/config/SecurityConfig.java

```diff
@@ -0,0 +1,36 @@
+package gov.cms.qpp.conversion.api.config;
+
+import gov.cms.qpp.conversion.api.security.JwtAuthorizationFilter;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.springframework.security.authentication.AuthenticationManager'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)